velox/type/Filter.h

```diff
@@ -618,6 +618,27 @@ class BigintRange final : public Filter {
         upper16_(std::min<int64_t>(upper, std::numeric_limits<int16_t>::max())),
         isSingleValue_(upper_ == lower_) {}
 
+  BigintRange(
+      int64_t lower,
+      bool lowerUnbounded,
```

Comment:
This change shouldn't be needed. 

lowerUnbounded can be implemented with lower set to std::numeric_limits<int64_t>::min()
lowerExclusive can be implemented by adjusting lower to lower + 1.

Ditto upperXxx.

Reply:
Thank you for the quick review. 
That's true. The reason of this change and https://github.com/facebookincubator/velox/pull/5815 is to create a unified API for different Ranges, so templates can be used when generating subfield filters. If they are not needed, we can change our implementations. Thanks again.

Reply:
It would be nice to keep existing APIs if possible.

Reply:
OK, we will resolve this difference in our usage.